tests/Unit/Helpers/Elliptic/FirstOrderSystem.hpp

```diff
@@ -0,0 +1,279 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Helper functions to test elliptic first-order systems
+
+#pragma once
+
+#include "Framework/TestingFramework.hpp"
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/PrefixHelpers.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "Elliptic/FirstOrderComputeTags.hpp"
+#include "Elliptic/FirstOrderOperator.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace TestHelpers::elliptic {
+
+template <typename System>
+void test_first_order_fluxes_computer(
+    const typename System::fluxes& fluxes_computer,
+    const DataVector& used_for_size) {
+  using FluxesComputer = typename System::fluxes;
+  static constexpr size_t volume_dim = System::volume_dim;
+  using vars_tag = typename System::fields_tag;
+  using primal_fields = typename System::primal_fields;
+  using auxiliary_fields = typename System::auxiliary_fields;
+  using VarsType = typename vars_tag::type;
+  using fluxes_tag =
+      db::add_tag_prefix<::Tags::Flux, vars_tag, tmpl::size_t<volume_dim>,
+                         Frame::Inertial>;
+  using argument_tags = typename FluxesComputer::argument_tags;
+
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> dist(0.5, 2.);
+
+  // Generate random variables
+  const auto vars = make_with_random_values<VarsType>(
+      make_not_null(&generator), make_not_null(&dist), used_for_size);
+
+  // Generate fluxes from the variables with random arguments
+  tuples::tagged_tuple_from_typelist<argument_tags> args{};
+  tmpl::for_each<argument_tags>(
+      [&args, &generator, &dist, &used_for_size](auto tag_v) {
+        using tag = tmpl::type_from<decltype(tag_v)>;
+        get<tag>(args) = make_with_random_values<typename tag::type>(
+            make_not_null(&generator), make_not_null(&dist), used_for_size);
+      });
+  const auto computed_fluxes = tuples::apply(
```

Comment:
is this `expected_fluxes`? I find having "computed" and "not computed" fluxes to be unclear as to what they mean.